pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <properties>
     <betonquest.version>2.0.0</betonquest.version>
-    <betonquest.version.qualifiers>${betonquest.version}-DEV-UNOFFICIAL</betonquest.version.qualifiers>
+    <betonquest.version.with.qualifier>${betonquest.version}-DEV-UNOFFICIAL</betonquest.version.with.qualifier>
```

Comment:
Why now with the word `with`?

Reply:
Because just `betonquest.version.qualifiers` sounds like a subset of `betonquest.version` which would be ONLY the qualifier. 
That is not the case, that's why I feel the need to clarify this by introducing `with`.

Reply:
i dont see the benefit of this. May the BQ version properties are a bit confusing. Because `betonquest.version` is more like the base version while `betonquest.version.with.qualifier` is the actual version, that get changed by the build pipeline as well